### ISSUE TITLE
errant inputs from snac port 1

### ISSUE THREAD

**Issue by Chudmega** [RobertPeip/Mister64#17]
i love this core, thanks for your work, just having an issue getting random inputs when using snac from port 1, ports 2-4 seem unaffected. only on 2024_03_03 build. 2024_03_01 build seems fine.

**Comment by blzla** [comment]
Can you give a bit more info on this issue? Does it happen in all games, or only in some or only in certain sections of those games. How frequent are the random issues? Do they happen when you don't touch the controller at all?

**Comment by Chudmega** [comment]
yes random inputs on all games while not touching controller, looks like any random button every couple seconds. using snax64 from misteraddons on mister multisystem from heber with original oem nintendo controller

**Comment by RobertPeip** [comment]
Several people in the discord channel, including the original SNAC code author blue1, tried to reproduce it and are not able.
The SNAC code internally is the same for all ports.
So the highest chance of something being broken is the controller or the adapter.
Both of course makes no sense if the old core does still work, but then we have no explanation.

**Comment by Chudmega** [comment]
after further testing, if its glitching it a power cycle might fix it but, it just seems very strange, previous core worked perfectly every time and when it is glitching its exacly as described, tried multiple cores, multiple controllers. i couldnt fix it before, and just switching cores would make it stable, thank you for checking it out and great work all around
